src/main/scala/org/hammerlab/guacamole/likelihood/Likelihood.scala

```diff
@@ -157,7 +157,7 @@ object Likelihood {
             case (allele, alleleFraction) =>
               alleleElementProbabilities(alleleToIndex(allele), ::) * alleleFraction
           }
-          sum( log( sum(alleleRows) )) + math.log(prior(genotype)) * depth
+          sum( log( sum(alleleRows) + 1e-10 )) + math.log(prior(genotype))
```

Comment:
OK, I've convinced myself that `sum(alleleRows) == 0` should be impossible unless there are what I'd call bugs elsewhere; what did you mean by "and all are empty" above?